src/sprites/rendered-target.js

```diff
@@ -724,6 +724,23 @@ class RenderedTarget extends Target {
         return null;
     }
 
+    /**
+     * Return whether touching the mouse, an edge, or a sprite.
+     * @param {string} requestedObject an id for mouse or edge, or a sprite name.
+     * @return {boolean} True if the sprite is touching the object.
+     */
+    touchingObject (requestedObject) {
```

Comment:
Nit: Can we change the name of this function to be 'isTouchingObject' since that indicates that this returns a boolean? Otherwise, this function name can also be interpreted as 'return whatever object this target is touching, or null if none'